app/models/user.rb

```diff
@@ -1,2 +1,9 @@
 class User < ApplicationRecord
+  has_many :likes
+  has_many :comments
+  has_many :posts
```

Comment:
- [ ] [REQUIRED] Good job so far! The `associations` between `models` should be set up correctly according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png). Please implement my suggestions below 🙏 

```ruby
# app/models/user.rb
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```